public/index.html

```diff
@@ -49,6 +49,15 @@
     -->
     <script type="text/javascript" src="https://cdn.fromdoppler.com/doppler-ui-library/v3.9.5/js/app.js"></script>
     <script type="text/javascript" src="zoho-chat.js"></script>
+    <script async src="https://www.googletagmanager.com/gtag/js?id=AW-1065197040/ZA62CKv_gZEBEPC79vsD"></script>
+    <script>
+      if (window.location.hostname !== 'localhost') {
```

Comment:
Why did you apply this check? 

I am asking because it will avoid tracking localhost, but it will still track our navigation in the CDN, is it fine?

Reply:
Maybe yes, because us use CDN for testing too, right?

Reply:
I do not know, I asked first!! :P